pysatSeasons/tests/test_occur_prob.py

```diff
@@ -23,7 +25,7 @@ def setup(self):
                                          clean_level='clean',
                                          orbit_info=orbit_info)
         self.testInst.bounds = (dt.datetime(2008, 1, 1),
-                                dt.datetime(2008, 1, 31))
+                                dt.datetime(2008, 1, 2))
```

Comment:
Recommend getting dates from test instrument instead of defining them here, also below.

Reply:
Turns out this requires the latest pysat, at least the use of `inst.inst_module._test_dates`

Reply:
It's easier with the newest pysat, but possible with the older ones by using the instrument module.